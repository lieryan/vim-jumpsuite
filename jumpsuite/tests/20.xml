<?xml version="1.0" encoding="utf-8"?>
<testsuites>
  <testsuite name="pytest" errors="2" failures="0" skipped="0" tests="28" time="38.686" timestamp="2022-12-13T17:07:59.079924" hostname="ryan-delta">
    <testcase classname="tests.test_python_images" name="test_mytest" time="4.257">
      <error message="failed on something">
Traceback (most recent call last):
  File "/home/user/myscript.py", line 5, in _foobar
    1 + foo(a,b,c=x['z']['x']['y']['z']['y'], d=e)
                  ~~~~~~~~~~~^^^^^
KeyError: 'y'</error>
    </testcase>
  </testsuite>
</testsuites>
