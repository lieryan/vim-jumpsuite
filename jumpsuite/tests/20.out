FAILED. Ran 28 tests in 38.686s (errors=2, failures=0, skipped=0)
/home/user/myscript.py:5:30: _foobar : KeyError: 'y' : 1 + foo(a,b,c=x['z']['x']['y']['z']['y'], d=e)
